snap/component.go

```diff
@@ -32,12 +32,13 @@ import (
 
 // ComponentInfo contains information about a snap component.
 type ComponentInfo struct {
-	Component           naming.ComponentRef `yaml:"component"`
-	Type                ComponentType       `yaml:"type"`
-	Version             string              `yaml:"version"`
-	Summary             string              `yaml:"summary"`
-	Description         string              `yaml:"description"`
-	ComponentProvenance string              `yaml:"provenance,omitempty"`
+	Component naming.ComponentRef `yaml:"component"`
+	Type      ComponentType       `yaml:"type"`
+	// YamlVersion should be used only in tests
+	YamlVersion         string `yaml:"version"`
```

Comment:
CompVersion would be more consistent with other naming conventions we use

Reply:
Changed